### PR TITLE
Cart implementation

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Implemented Cart Functionality.
-> Handled the responsiveness .

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.App {
+  position: relative;
+}
+/* ::-webkit-scrollbar {
```
Comment (Prabhav-Sharma) [inline src/App.css]: Remove comments if they're not needed

File: src/pages/Cart/Cart.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* .page-container {
+  height: calc(100vh - 4rem);
+  overflow-y: scroll;
+  scroll-behavior: smooth;
+  width: 100vw;
+  direction: ltr;
```
Comment (Prabhav-Sharma) [inline src/pages/Cart/Cart.css]: Remove comment

File: src/just.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Prabhav-Sharma) [inline src/just.html]: What does this file do?

File: src/pages/Cart/Cart.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import "./Cart.css";
+import { useUserData } from "../../context/UserDataContext";
+import { Horizontal_card } from "../../Components";
+import { price_of_cart } from "../../utils";
+
+export const Cart = () => {
+  const { user_data } = useUserData();
+  let cart_price = price_of_cart(user_data.cart);
+  return (
+    <div class="page-container">
```
Comment (Prabhav-Sharma) [inline src/pages/Cart/Cart.js]: use className in jsx instead of class

File: src/utils/price_of_cart/price_of_cart.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export const price_of_cart = (cart) => {
+  let price = cart.reduce(
+    (accum, ele) => (accum += Number(ele.price) * Number(ele.qnty)),
```
Comment (Prabhav-Sharma) [inline src/utils/price_of_cart/price_of_cart.js]: Use proper names for object properties to avoid typos and confusion

**Review by Prabhav-Sharma — CHANGES_REQUESTED** [review]
Overall the code is good, I've suggested a few minor changes.

File: src/App.css (hunk 1 above)
Comment (ashishnnnnn) [reply src/App.css]: Done..

File: src/pages/Cart/Cart.css (hunk 2 above)
Comment (ashishnnnnn) [reply src/pages/Cart/Cart.css]: Done ..

File: src/just.html (hunk 3 above)
Comment (ashishnnnnn) [reply src/just.html]: It was to be removed..

File: src/utils/price_of_cart/price_of_cart.js (hunk 5 above)
Comment (ashishnnnnn) [reply src/utils/price_of_cart/price_of_cart.js]: Changed with semantic name

File: src/pages/Cart/Cart.js (hunk 4 above)
Comment (ashishnnnnn) [reply src/pages/Cart/Cart.js]: Done

**Review by Prabhav-Sharma — APPROVED** [review]

**Review by Vishruta-Patil — APPROVED** [review]
Clean and well-structured Code. I liked the way you have added items to the local storage.
Suggestion - Add the details of the features implemented in the PR description, it will help in code review

**Pull request merged**